learn/configuration/instance_options.md

```diff
@@ -145,12 +155,17 @@ If no master key is provided in a `development` environment, all routes will be
 
 ### Disable analytics
 
+::: warning
+This is a CLI flag and does not take any values. Assigning a value will throw an error.
+:::
+
 **Environment variable**: `MEILI_NO_ANALYTICS`
 **CLI option**: `--no-analytics`
-**Default value**: `false`
-**Expected value**: `true` or `false`
+**Default**: Enabled
+
+Deactivates Meilisearch's built-in telemetry when enabled.
 
-Deactivates Meilisearch's built-in telemetry when set to `true`.
+The environment variable accepts `n`, `no`, `f`, `false`, `off`, and `0` as false literals. An absent environment variable will also be considered as false, everything else is considered `true`.
```

Comment:
Do you think "false literals" might be too technical here? Perhaps we should rephrase it to something simpler?